Cluster processing

alex-golub99 opened the pull request:
Added cluster processing in convert_to_training_data function. Includes: rescaling pT, flipping eta, cluster energy fraction

gordonwatts commented on calratio_training_data/constants.py:
```diff
@@ -20,3 +20,7 @@
 LLP_Lz_max = 6000
+
+# Min/Max Jet pT 
+min_jet_pt = 40 # GeV
```
Please make sure to run `black` and `flake8` on these. I should have that as part of testing!
```suggestion
min_jet_pt = 40  # GeV
```

alex-golub99 commented:
Added the clarification for the min/max jet pT constants and moved the processing code to a new file and just called it in the main function. Added a flag in that function for doing rotations, default true. Need to still make a more global flag for whether or not we do rotations, and still need to add tests.

alex-golub99 commented:
Added the actual rotation code (this was missing before). Still need to make test cases to double check it's working correctly (though it does run).
